src/diskquota.c

```diff
@@ -1860,10 +1865,11 @@ static void
 vacuum_db_entry(DiskquotaDBEntry *db)
 {
 	if (db == NULL) return;
-	db->dbid     = InvalidOid;
-	db->inited   = false;
-	db->workerId = INVALID_WORKER_ID;
-	db->in_use   = false;
+	db->dbid      = InvalidOid;
+	db->inited    = false;
+	db->workerId  = INVALID_WORKER_ID;
+	db->in_use    = false;
+	db->corrupted = false;
```

Comment:
It seems that only one line was added here, but for some reason four have changed!

Reply:
Otherwise, CI check for code style fails:
git ls-files *.{c,h} | xargs clang-format -i --style=file && git diff --exit-code